### ISSUE TITLE
Training set, validation set and testing set

### ISSUE THREAD

**Issue by JamenceTom** [chufengt/iccv19_attribute#66]
Hi! It is a great job.
In your paper, I notice that PA-100K dataset is divided into training set, validation set and testing set. However, training set and validation set are combined together to train the model in this implementation. In pedestrian attribute recognition task, is it normal to train the model with training set and validation set?
I am just confused about it. Thank you very much.

**Comment by chufengt** [comment]
Sorry for the late reply.  For this problem, we simply follow the split principle of RAP that only distinguishes the training and testing sets.  Maybe the comparison is not totally fair while most previous works did not release the code. I think our model can still surpass previous methods even without the val set (10k images for PA-100K).
